packages/fets/src/client/createClient.ts

```diff
@@ -107,6 +107,7 @@ export function createClient({ endpoint, fetchFn = fetch, plugins = [] }: Client
             const requestInit: RequestInit & { headers: Record<string, string> } = {
               method,
               headers: requestParams?.headers || {},
+              signal: init?.signal,
```

Comment:
```suggestion
              ...init,
```
Maybe this is better to accept other parameters as well.

Reply:
Updated in 8b98936